containers/azure-terraform/README.md

```diff
@@ -15,19 +17,24 @@
 
 ## Using this definition with an existing folder
 
-While technically optional, this definition includes the Azure Terraform extension which requires an Azure account to use. You can create a [free trial account here](https://azure.microsoft.com/en-us/free/) and find out more about using [Teraform with Azure here](https://docs.microsoft.com/en-us/azure/terraform/terraform-overview).  
+While technically optional, this definition includes the Azure Terraform extension which requires a few pre-requisites:
+
+- [Azure Subscription](https://azure.microsoft.com): A current Azure subscription or a free trail account needed.
```

Comment:
Nit: spelling of trial.  Also, should we link the words "free trial" to https://azure.microsoft.com/en-us/free/

Reply:
Addressed, thank you!